dockers/docker-fpm-frr/base_image_files/TSB

```diff
@@ -57,10 +57,4 @@ if [ -z "$STARTED_BY_TSA_TSB_SERVICE" ]; then
 fi
 
 /usr/bin/TS TSB
-if [[ "$(sonic-cfggen -d -v DEVICE_METADATA.localhost.type)" == *"SpineRouter"* ]] ; then
-  if [[ "$1" != "chassis" ]] ; then
-    echo "Please execute 'sudo config save' to preserve System mode in Normal state after reboot or config reload"    
-  fi
-else
-  echo "Please execute 'sudo config save' to preserve System mode in Normal state after reboot or config reload"
-fi
```

Comment:
why remove this ?

Reply:
This is some previous code cleanup issue. It is the same message being printed in all cases, so removed the conditional check